gen_cert.py

```diff
@@ -1046,8 +338,6 @@ def _generate_verification_page(self, name, filename, output_dir, verify_uuid, d
             return
 
         prefix = ''
-        if self.template_version == 2:
-            prefix = 'v2/'
```

Comment:
I don't think `prefix` is being used any longer.
Can we remove it above and below?